### PR TITLE
Adding note about admin permissions

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2553
@JoanneHendrickson  Please kindly review

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/Fileshare-to-ODSP-migration-guide.md [hunk 1]
```diff
@@ -119,6 +119,8 @@ Before you migrate your file share content, you must pre-provision your users in
 
 The following is a typical migration process that follows Microsoft's best practices guidance.
 
+> [!IMPORTANT] Make sure that the account used to migrate content has permissions on the destination OneDrive.
```
Comment (JohanFreelancer9) [inline migration/Fileshare-to-ODSP-migration-guide.md:122]:
```suggestion
> [!IMPORTANT]
> Make sure that the account used to migrate content has permissions on the destination OneDrive.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by Marweis** [comment]
Hi Folks, Hoping to get this approved today before a meeting with a Cx. Please apply if possible. Thank you!

**Comment by get-itips** [comment]
@JoanneHendrickson Please review. Thank you!

**Pull request merged**